docs/cli/edgedb_server_destroy.rst

```diff
@@ -0,0 +1,33 @@
+.. _ref_cli_edgedb_server_destroy:
+
+
+=====================
+edgedb server destroy
+=====================
+
+Stop an EdgeDB instance.
+
+.. cli:synopsis::
+
+     edgedb server destroy [OPTIONS] <name>
+
+
+Description
+===========
+
+``edgedb server destroy`` is a terminal command for destroying an
+EdgeDB instance. The instance must be :ref:`stopped
```

Comment:
Instance doesn't have to be stopped first.

Reply:
Fixed